submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/service/FileSystemService.kt

```diff
@@ -10,33 +9,19 @@ import mu.KotlinLogging
 private val logger = KotlinLogging.logger {}
 
 class FileSystemService(
-    private val ftpService: FtpService,
     private val filesService: FilesService,
     private val pageTabService: PageTabService,
 ) {
     fun persistSubmissionFiles(request: FilePersistenceRequest): ExtSubmission {
-        val (submission, mode) = request
-        val accNo = submission.accNo
-        val owner = submission.owner
-
-        logger.info { "$accNo $owner Processing files of submission $accNo in mode $mode" }
+        val (sub, mode) = request
+        logger.info { "${sub.accNo} ${sub.owner} Processing files of submission ${sub.accNo} in mode $mode" }
```

Comment:
I think we should create a ticket to deprecate the "mode" since it doesn't really add any value now that we're working on FIRE

Reply:
still plenty submissions to migrate :(